docs/conf.py

```diff
@@ -316,7 +316,11 @@ def linkcode_resolve(domain, info):
 # html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-# html_sidebars = {}
+
+html_sidebars = {
+    "**": ["sidebar-nav-bs", "sidebar-ethical-ads"],
+    "documentation/additional_examples": []
```

Comment:
So no additional examples yet?

Reply:
This change is only to do with the sidebars themselves. `"**": ["sidebar-nav-bs", "sidebar-ethical-ads"]` is the default setting, showing the page navigation (left) and the section navigation (right). On the additional_examples page there was horizontal scrolling which made it difficult to read the code, so I disabled the sidebars.